app/assets/stylesheets/view_specific/shared/email_status.scss

```diff
@@ -1,4 +1,4 @@
-.email-status {
+.view-shared-email_status {
```

Comment:
My concern here is that this will end up as a long class name. I like the idea of a single letter prefix. Some options:

- v-email_status - this would be dropping the shared prefix, which might be ok.
- vs-email_status 
- v-s-email_status